README.md

```diff
@@ -133,16 +134,6 @@ Stage some changes first.
 git add .
 ```
 
-## macOS Issues
-
-❌ timeout: command not found
-
-Install GNU coreutils
-
-```sh
-brew install coreutils
-```
```

Comment:
No one will ever get this error since you updated the script to check for `timeout`.